libs/qrencode/Makefile

```diff
@@ -56,8 +66,8 @@ digits or 4000 characters, and is highly robust.
 endef
 
 CMAKE_OPTIONS += \
-	-DCMAKE_DISABLE_FIND_PACKAGE_PNG=ON \
-	-DWITHOUT_PNG=ON \
+	-DCMAKE_DISABLE_FIND_PACKAGE_PNG=O$(if $(CONFIG_QRENCODE_PNG),FF,N) \
+	-DWITHOUT_PNG=O$(if $(CONFIG_QRENCODE_PNG),FF,N) \
```

Comment:
need PKG_CONFIG_DEPENDS I think.

Reply:
I'm not exactly sure what that does and how to write it.
Is this ok?
`PKG_CONFIG_DEPENDS:=CONFIG_PACKAGE_qrencode CONFIG_PACKAGE_libpng`
If I write it like that, will it not trigger libpng to be built even when PNG option is off?

Reply:
Not quite. You need to write the name of the option itself in terms of config to force rebuilds. There are examples in this repo.

Reply:
Not helpful.

Reply:
uhhhh OK

Look at your .config file and look for QRENCODE_PNG . Use the full string.

Reply:
`PKG_CONFIG_DEPENDS:=CONFIG_PACKAGE_qrencode CONFIG_QRENCODE_PNG` ?

Reply:
It's probably the first option.
Added.